docs/geometry.rst

```diff
@@ -145,15 +145,15 @@ A format specification may also be used to control the format and precision.
     or hex-encoded as 0101000000cf6a813d263d65c0bdaab35a60ff32c0
 
 Shapely has a format specification inspired from Python's
-`Format Specification Mini-Language
-<https://docs.python.org/3/library/string.html#formatspec>`_ described next.
+:ref:`formatspec`, described next.
```

Comment:
When reading/editing this source, it might not be directly clear this is an external reference, so we could explicitly "namespace" it, like the following?

```suggestion
:ref:`python:formatspec`, described next.
```